doc/howtos/booting-and-installation.rst

```diff
@@ -127,6 +127,58 @@ includes a checksum verification, you can also use the traditional :command:`dd`
 Unplug the removable media from your development system and you're ready to plug
 it into your target system.
 
+Installing to internal media on generic x86 EFI platforms
+=========================================================
+
+This section applies to those cases where the device is x86 compatible (ex: Gigabyte NUC).
+The simpler way is to follow these steps:
+
+#. Create a Bootable Media using an both an image that is compatible with the device
+   and a supported removable media (SD card or USB stick).
+#. Boot the device with the removable media created at the previous step.
+#. Once the device is booted, identify the booted media, for example by executing::
+
+      $ mount
+
+   and locating in its output which partition is mounted ``on /``.
+   This will be a partition belonging to the source block device.
+   Example::
+
+      /dev/sdb3 on / type ext4 (rw,realtime,i_version,data=ordered)
+
+   Shows that the root partition is on `/dev/sdb3`, therefore the source block device will
+   be `/dev/sdb`.
+#. Identify the block device representing the internal storage.
+   This part is really specific to each platfrom: typically the internal storage is
+   associated to `/dev/sda` for hard disks and SSD units and to `/dev/mmcblk0` for eMMC
+   devices, but it should be confirmed against the board manual.
+#. Issue the `dd` command with the parameters identified in the last 2 points.
+   Example with internal storage as eMMC on `/dev/mmcblk0` and removable media as USB stick
+   on `/dev/sda`::
+
+      $ dd if=/dev/sda of=/dev/mmcblk0 bs=5M && sync  
+
+#. When the command has terminated, poweroff the device, extract the removable media
+   and power it back on. This should be sufficient to have the unit now using the internal
+   storage.
+
+
+Removing Ostro OS from internal media
+=====================================
+   Ostro OS uses GID for identifying the rootfs and the GID is kept consistent across all the
+   images produced. This means that trying to boot a system with 2 Ostro OC images avilable at the
+   same time will likely produce unwanted/undetermined results.
+   Before trying again to boot from a removable media, the internal media should be wiped.
+   To achieve this:
+#. Boot from the internal media.
+#. Identify the root block device containing the rootfs partition.
+#. Wipe a sufficient part of that block device.
```

Comment:
Can you perhaps expand on what 'sufficient' means? In the command below, I think what you're doing is essentially wipe the GPT and that's it, is it correct?